lib/usd/hdMaya/adapters/materialAdapter.h

```diff
@@ -43,7 +43,14 @@ class HdMayaMaterialAdapter : public HdMayaAdapter
     HDMAYA_API
     void Populate() override;
 
-#if USD_VERSION_NUM <= 1911
+#if USD_VERSION_NUM <= 2011
+
+    HDMAYA_API
+    virtual HdTextureResourceSharedPtr GetTextureResource(const SdfPath& textureShaderId);
+    HDMAYA_API
+    virtual HdTextureResource::ID GetTextureResourceID(const TfToken& paramName);
+
+#elif USD_VERSION_NUM <= 1911
```

Comment:
this elif branch will never be entered - see above note

Reply:
Addressed by d9ea7b2abc0f27f79638e3602c129a9812e73fd1.